src/components/account.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom'
+
+import { StyledBtns, StyledNav } from '../Styles';
+
+export default function Account() {
+   return (
+      <>
+         <header className="header">
+            <h1>Randomized Character Creator</h1>
+
+            <nav>
```

Comment:
If this nav is going to be used everywhere you could make this it's own component to use